src/04-modality-specific-files/05-task-events.md

```diff
@@ -33,16 +33,15 @@ Each task events file REQUIRES a corresponding task imaging data file
+| onset           | REQUIRED              | [number][]    | Onset (in seconds) of the event measured from the beginning of the acquisition of the first volume in the corresponding task imaging data file. If any acquired scans have been discarded before forming the imaging data file, ensure that a time of 0 corresponds to the first image stored. In other words negative numbers in "onset" are allowed<sup>5</sup>. |
+| duration        | REQUIRED              | [number][]    | Duration of the event (measured from onset) in seconds. Must always be either zero or positive. A "duration" value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse.                                                                                                                                        |
+| sample          | OPTIONAL              | [number][]    | Onset of the event according to the sampling scheme of the recorded modality (that is, referring to the raw data file that the `events.tsv` file accompanies).                                                                                                                                                                                                     |
+| trial_type      | OPTIONAL              | [string][]    | Primary categorisation of each trial to identify them as instances of the experimental conditions. For example: for a response inhibition task, it could take on values "go" and "no-go" to refer to response initiation and response inhibition experimental conditions.                                                                                          |
+| response_time   | OPTIONAL              | [number][]    | Response time measured in seconds. A negative response time can be used to represent preemptive responses and "n/a" denotes a missed response.                                                                                                                                                                                                                     |
+| value           | OPTIONAL              | ????          | Marker value associated with the event (for example, the value of a TTL trigger that was recorded at the onset of the event).                                                                                                                                                                                                                                      |
+| HED             | OPTIONAL              | ????          | Hierarchical Event Descriptor (HED) Tag. See [Appendix III](../99-appendices/03-hed.md) for details.                                                                                                                                                                                                                                                               |
```

Comment:
Not sure if HED have a specific datatype that should be mentioned here.

tagging @VisLab

Reply:
HED tags are usually strings, but I'll let Kay confirm just to be sure.

Reply:
@VisLab what would be the datatype for the HED column in an `event.tsv`

Reply:
as reported here: https://github.com/bids-standard/bids-specification/pull/697#issuecomment-789128879, this should be a string